content/learn/book/migration-guides/0.7-0.8/_index.md

```diff
@@ -0,0 +1,399 @@
+}
+```
+
+After:
+
+```rust
+fn parallel_system(query: Query<&MyComponent>) {
+   query.par_for_each(32, |comp| {
+        ...
+   });
+}
+```
+
+If using `Query(State)` outside of a system run by the scheduler, you may need to manually configure and initialize a `ComputeTaskPool` as a resource in the `World`.
+
+### [Fail to compile on 16-bit platforms](https://github.com/bevyengine/bevy/pull/4736)
+
+`bevy_ecs` will now explicitly fail to compile on 16-bit platforms. If this is required, there is currently no alternative. Please file an issue (<https://github.com/bevyengine/bevy/issues>) to help detail your use case.
+
+### [Camera Driven Rendering](https://github.com/bevyengine/bevy/pull/4745)
+
+<!-- TODO pls cart add a migration guide section :( -->
+
+This is a very complicated change and it is recommended to read the linked PRs for more details
+
+```rust
+// old 3d perspective camera
+commands.spawn_bundle(PerspectiveCameraBundle::default())
+
+// new 3d perspective camera
+commands.spawn_bundle(Camera3dBundle::default())
+```
+
+```rust
+// old 2d orthographic camera
+commands.spawn_bundle(OrthographicCameraBundle::new_2d())
+
+// new 2d orthographic camera
+commands.spawn_bundle(Camera2dBundle::default())
+```
+
+```rust
+// old 3d orthographic camera
+commands.spawn_bundle(OrthographicCameraBundle::new_3d())
+
+// new 3d orthographic camera
+commands.spawn_bundle(Camera3dBundle {
+    projection: OrthographicProjection {
+        scale: 3.0,
+        scaling_mode: ScalingMode::FixedVertical,
+        ..default()
+    }.into(),
+    ..default()
+})
+```
+
+UI no longer requires a dedicated camera. `UiCameraBundle` has been removed. `Camera2dBundle` and `Camera3dBundle` now both default to rendering UI as part of their own render graphs. To disable UI rendering for a camera, disable it using the CameraUi component:
+
+```rust
+commands
+    .spawn_bundle(Camera3dBundle::default())
+    .insert(UiCameraConfig {
+        show_ui: false,
+        ..default()
+    })
+```
+
+### [Enforce type safe usage of Handle::get](https://github.com/bevyengine/bevy/pull/4794)
+
+`Assets::<T>::get` and `Assets::<T>::get_mut` now require that the passed handles are `Handle<T>`, improving the type safety of handles. If you were previously passing in:
+
+* a `HandleId`, use `&Handle::weak(id)` instead, to create a weak handle. You may have been able to store a type safe `Handle` instead.
+* a `HandleUntyped`, use `&handle_untyped.typed_weak()` to create a weak handle of the specified type. This is most likely to be the useful when using [load_folder](https://docs.rs/bevy_asset/latest/bevy_asset/struct.AssetServer.html#method.load_folder)
+* a `Handle<U>` of  of a different type, consider whether this is the correct handle type to store. If it is (i.e. the same handle id is used for multiple different Asset types) use `Handle::weak(handle.id)` to cast to a different type.
+
+### [Allow higher order systems](https://github.com/bevyengine/bevy/pull/4833)
```

Comment:
Ping @DJMcNab. I can't see any breaking behavior in a cursory pass.

Reply:
It was *technically* a breaking change, since it changes the public API of `SystemParamFunction`.
I was expecting pushback against it being treated as 'not a breaking change', so I marked it as such to be correct.

Maybe a note that `SystemParamFunction` has been changed, in order to allow composing systems? I still think that there have been no external users, so we can probably just remove this section.